<!-- -->
<launch>
  <arg name="output_frame" default="lidar_frame" />
  <arg name="input0" default="/lidar0/points_raw" />
  <arg name="input1" default="/lidar1/points_raw" />
  <arg name="output" default="/points_concat" />

  <node pkg="points_preprocessor" type="points_concat_filter"
        name="points_concat_filter" output="log">
    <param name="output_frame" value="$(arg output_frame)" />
    <remap from="/lidar0/points_raw" to="$(arg input0)" />
    <remap from="/lidar1/points_raw" to="$(arg input1)" />
    <remap from="/points_concat" to="$(arg output)" />
  </node>
</launch>
